Documentation for temporary tables (#702)
vitessio/website: content/en/docs/reference/compatibility/mysql-compatibility.md

CREATE PROCEDURE is not supported. You have to create the procedure directly on the underlying MySQL servers and not through Vitess.

### Temporary Tables

> Vitess has a limited support for temporary tables. It works only for unsharded keyspace.

Reviewer: ```suggestion
Vitess has limited support for temporary tables. It works only for unsharded keyspaces.
```

> If the user create a temporary table then the session will start using reserved connections for any query sent on that session 

Reviewer: ```suggestion
If the user creates a temporary table then the session will start using reserved connections for any query sent on that session.
```

> and also the plans generated by those will not be caches. But, they will continue to use the plan caches from other sessions.  

Reviewer: ```suggestion
The plans generated by those queries will not be cached but they will continue to use the plans cached from other sessions.  
```
